packages/shared/lib/contexts/onboarding/actions/initialiseSecretManager.ts

```diff
@@ -0,0 +1,19 @@
+import { get } from 'svelte/store'
+
+import { ProfileRecoveryType, ProfileSetupType } from '../enums'
+import { onboardingProfile } from '../stores'
+
+import { restoreBackupFromStrongholdFile } from './restoreBackupFromStrongholdFile'
+
+export async function initialiseSecretManager(): Promise<void> {
```

Comment:
name doesn't make sense based on the functionality

Reply:
What doesn't make sense about it? Maybe `initialiseSecretManagerForOnboardingProfile`? As of now it only includes initialization logic for stronghold restoration, but this function shouldn't even be needed anyway.